dev/package-beats/aerospike-0.0.1/manifest.yml

```diff
@@ -0,0 +1,12 @@
+format_version: 1.0.0
+name: aerospike
+version: 0.0.1
+description: ""
+type: integration
+categories:
+- metrics
+requirement:
+  kibana:
+    versions: ""
+download: ""
```

Comment:
We should find a way that download and path don't show up here. They are not needed in the manifest.

Reply:
At the moment I can mark them with `omitempty` to simply skip them if empty. If this is insufficient I can clean their content before writing to file.